basis/configuration/graph.py

```diff
@@ -85,6 +85,4 @@ class ExposingCfg(FrozenPydanticBase):
 class GraphDefinitionCfg(FrozenPydanticBase):
     name: str = None
     exposes: ExposingCfg = None
-    description: str = None
```

Comment:
These fields were never read, and weren't present in the manifest. We can add them back in a future PR if we end up wanting them

Reply:
👍